README.md

```diff
@@ -18,97 +18,39 @@
-<kbd>[<img title="Italiano" alt="Italiano" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/it.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_it_IT.md)</kbd>
-<kbd>[<img title="ᜆᜄᜎᜓᜄ᜔ (Wikang Tagalog)" alt="ᜆᜄᜎᜓᜄ᜔ (Wikang Tagalog)" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/ph.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_ph_PH.md)</kbd>
-<kbd>[<img title="Polski" alt="Polski" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/pl.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_pl_PL.md)</kbd>
-<kbd>[<img title="Português (Brasil)" alt="Português (Brasil)" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/br.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_pt_BR.md)</kbd>
-<kbd>[<img title="Romania" alt="Romania" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/ro.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_ro_RO.md)</kbd>
-<kbd>[<img title="Русский" alt="Русский" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/ru.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_ru_RU.md)</kbd>
-<kbd>[<img title="Svenska" alt="Svenska" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/se.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_sv_SE.md)</kbd>
-<kbd>[<img title="Türkçe" alt="Türkçe" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/tr.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_tr_TR.md)</kbd>
-<kbd>[<img title="ไทย" alt="ไทย" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/th.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_th_TH.md)</kbd>
-<kbd>[<img title="Українська" alt="Українська" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/ua.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_ua_UA.md)</kbd>
-<kbd>[<img title="Tiếng Việt" alt="Tiếng Việt" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/vn.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_vi_VN.md)</kbd>
-<kbd>[<img title="中文" alt="中文" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/cn.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_zh_CN.md)</kbd>
-#### _Do you want to help us by translating documents? If so, please follow these [instructions](https://github.com/Atlas-OS/Atlas/blob/main/translations/README.md)._
-
 ## 📚 **Table of contents**
```

Comment:
Probably should be changed to something like `Important Documentation` instead of ToC, as it looks like it's the ToC for the README instead of the documentation. Plus, that isn't the entire ToC for the documentation anyways.